odoo/modules/module.py

```diff
@@ -524,3 +524,12 @@ def check_manifest_dependencies(manifest: dict) -> None:
             tools.find_in_path(binary)
         except OSError:
             raise Exception('Unable to find %r in path' % (binary,))
+
+
+def load_script(path: str, module_name: str):
+    full_path = file_path(path) if not os.path.isabs(path) else path
```

Comment:
I think you should always use `file_path` due to security checks it does.

Reply:
I don't know, this code has explicitly been added in 2019 by CHS and ODO for some reason dealing with the upgrade path 🤔 ...
https://github.com/odoo/odoo/commit/bbb1a8f151b07d0125fc6d7d89375aa22681aac5#diff-4f4d1294e4370c88e155c98b6227e7a7991a9f2ea656e1df569fa78a16091518R22

![immagine](https://github.com/user-attachments/assets/7be256be-df27-4915-8392-7227c5a0c931)